tests/tests.go

```diff
@@ -194,6 +216,89 @@ func DeleteLeaderManager() {
 	time.Sleep(40 * time.Second)
 }
 
+func changeManagerReplicas(numOfReplica int32) error {
+	Eventually(func() error {
+		managerDeployment, err := testClient.KubeClient.AppsV1().Deployments(ManagerNamespce).Get(names.MANAGER_DEPLOYMENT, metav1.GetOptions{})
+		if err != nil {
+			return err
+		}
+
+		managerDeployment.Spec.Replicas = &numOfReplica
```

Comment:
I'm not sure, but if numOfReplica is int32, then why are we using reference operator?

Reply:
Type of Replicas is *int32 to allow nil as the default AFAIK

Reply:
yeah my bad thanks.